src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,20 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new java.io.FileReader(filePath))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                lines.add(line);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
Instead of printing the stack trace, you should rethrow the exception as a RuntimeException with an informative message. For example: `throw new RuntimeException("Cannot read file: " + filePath, e);`